buidl/test/test_hd.py

```diff
@@ -404,7 +404,8 @@ def test_bech32_address(self):
     def test_zprv(self):
         mnemonic, priv = HDPrivateKey.generate(extra_entropy=1 << 128)
         for word in mnemonic.split():
-            self.assertTrue(word in WORD_LIST)
+            # should throw an error if this is not a word
+            BIP39.index(word)
```

Comment:
Repeated point from before, but using a method like this makes it non-obvious that it should throw an error if this is not a word, whereas a normal dictionary lookup doesn't have this ambiguity.

Reply:
yea, i'll use `__getitem__` so it works like a dictionary lookup.